pkg/openshift/rosa/cluster.go

```diff
@@ -131,20 +131,18 @@ func (r *Provider) CreateCluster(ctx context.Context, options *CreateClusterOpti
 			return "", &clusterError{action: action, err: err}
 		}
 		options.accountRoles = *accountRoles
-	}
 
-	if options.HostedCP {
-		if options.OidcConfigID == "" {
```

Comment:
Can we leave this condition? That way a user can still provide in their own OidcConfigID if desired